lib/board.rb

```diff
@@ -0,0 +1,49 @@
+class Board
+attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")                                                                                            
+  end
+
+  def cells
+    @cells
+  end
+
+  def display
+    puts " "
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+    puts " "
+  end
+
+  def position(pos)
+    @cells[pos.to_i-1]
+  end
+
+  def taken?(pos)
+    position(pos) == "X" || position(pos) == "O" ? true : false
```

Comment:
Use the implicit boolean return value of the expression instead of adding the ternary. 

`position(pos) == "X" || position(pos) == "O"`